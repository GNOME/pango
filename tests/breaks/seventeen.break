<span lang='ca-es'>És una prova d'instal·lació</span>
<span lang='ca-fr'>És una prova d'instal·lació</span>
<span lang='en-US'>És una prova d'instal·lació</span>
