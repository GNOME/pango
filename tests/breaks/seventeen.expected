Text:         ⁦É⁩ ⁦s⁩  [ ]  ⁦u⁩ ⁦n⁩ ⁦a⁩  [ ]  ⁦p⁩ ⁦r⁩ ⁦o⁩ ⁦v⁩ ⁦a⁩  [ ]  ⁦d⁩ ⁦'⁩ ⁦i⁩ ⁦n⁩ ⁦s⁩ ⁦t⁩ ⁦a⁩ ⁦l⁩ ⁦·⁩ ⁦l⁩ ⁦a⁩ ⁦c⁩ ⁦i⁩ ⁦ó⁩  [0x0a]  ⁦É⁩ ⁦s⁩  [ ]  ⁦u⁩ ⁦n⁩ ⁦a⁩  [ ]  ⁦p⁩ ⁦r⁩ ⁦o⁩ ⁦v⁩ ⁦a⁩  [ ]  ⁦d⁩ ⁦'⁩ ⁦i⁩ ⁦n⁩ ⁦s⁩ ⁦t⁩ ⁦a⁩ ⁦l⁩ ⁦·⁩ ⁦l⁩ ⁦a⁩ ⁦c⁩ ⁦i⁩ ⁦ó⁩  [0x0a]  ⁦É⁩ ⁦s⁩  [ ]  ⁦u⁩ ⁦n⁩ ⁦a⁩  [ ]  ⁦p⁩ ⁦r⁩ ⁦o⁩ ⁦v⁩ ⁦a⁩  [ ]  ⁦d⁩ ⁦'⁩ ⁦i⁩ ⁦n⁩ ⁦s⁩ ⁦t⁩ ⁦a⁩ ⁦l⁩ ⁦·⁩ ⁦l⁩ ⁦a⁩ ⁦c⁩ ⁦i⁩ ⁦ó⁩  [0x0a]  
Breaks:     c  c c    lc c c c    lc c c c c c    lc c c c c c c c c c c c c c c       Lc c c    lc c c c    lc c c c c c    lc c c c c c c c c c c c c c c       Lc c c    lc c c c    lc c c c c c    lc c c c c c c c c c c c c c c       Lc
Whitespace:      x           x               x                                 w            x           x               x                                 w            x           x               x                                 w       w 
Sentences:  bs                                                                 e       bs                                                                 e       bs                                                                 e       b 
Words:      bs   be   bs     be   bs         be   bs e s                       be      bs   be   bs     be   bs         be   bs e s                       be      bs   be   bs     be   bs         be   bs e s           e s         be      b 
Graphemes:  b  b b    b  b b b    b  b b b b b    b  b b b b b b b b b b b b b b       b  b b    b  b b b    b  b b b b b    b  b b b b b b b b b b b b b b       b  b b    b  b b b    b  b b b b b    b  b b b b b b b b b b b b b b       b 
Hyphens:       i         i i         i i i i             i i i i i     i i i i            i         i i         i i i i             i i i i i     i i i i            i         i i         i i i i             i i i i i     i i i i           
